opendrift/models/basemodel.py

```diff
@@ -2357,15 +2357,17 @@ def run(self, time_step=None, steps=None, time_step_output=None,
 
             self.timer_end('preparing main loop:making dynamical landmask')
 
-        # Move point seeded on land to ocean
-        if self.get_config('seed:ocean_only') is True and \
-            ('land_binary_mask' in self.required_variables):
-            #('land_binary_mask' not in self.fallback_values) and \
-            self.timer_start('preparing main loop:moving elements to ocean')
-            self.elements_scheduled.lon, self.elements_scheduled.lat = \
+
+        # Move point seed on land to ocean
+        if move_from_land==True:
+            if self.get_config('seed:ocean_only') is True and \
+                ('land_binary_mask' not in self.fallback_values) and \
+                ('land_binary_mask' in self.required_variables):
+                self.timer_start('preparing main loop:moving elements to ocean')
+                self.elements_scheduled.lon, self.elements_scheduled.lat = \
```

Comment:
It seems here that you have simply overwritten the config `seed:ocean_only` with the new `move_from_land=True`, and this is probably one (of several?) reason for the tests failing.
Anyway, if starting from a fresh, rebased fork, it should be quite straightforward to make a new PR with the same additions as in this PR. So two things are essential: rebasing, and running pytest before committing.